packages/lesswrong/components/editor/CKPostEditor.tsx

```diff
@@ -445,7 +459,7 @@ const CKPostEditor = ({
 
   return <div>
     {isBlockOwnershipMode && <>
-     <DialogueEditorGuidelines />
+     {!hasEverDialoguedBefore && <DialogueEditorGuidelines />}
```

Comment:
My guess is that this causes the box to not show up for a bit while the page is loading (if the user navigates to it from elsewhere on LW), while the query is still in a loading state.

Reply:
yeah. suggested better way of doing it?

Reply:
hm.. not actually sure there's a good solution here... can't rely on the query's `loading`, since either way you don't know whether you should be showing it ahead of time or not.  it's probably fine; the query should be pretty fast itself.